Fix config (#16)
kintone/SAMPLE-Date-input-button-Plug-in: README.md

2. Make changes to files under /src
3. Repackage the plug-in by:  
 i. Zipping the manifest.json file, css directory, html directory, image directory and js directory into one zip file.  

>  ii. Drag and dropping the file into the [kintone plug-in packer](https://plugin-packer.kintone.dev/).

Reviewer: ```suggestion
 ii. Packaging the file using [kintone plug-in packer](https://plugin-packer.kintone.dev/).
```
